src/utils/alerts.js

```diff
@@ -0,0 +1,41 @@
+import { store } from '..'
+import { clearAlert, clear } from '../actions/status'
+
+export const showAlert = ({data, onlyError, onlySuccess, console = true, localAlert, messageCodeHeader} = {}) => ({
+    alert: {
+        showAlert: localAlert ? false : true,
+        onlyError: onlySuccess ? false : true,
+        onlySuccess: onlyError ? false : true,
+        console,
+        localAlert,
+        messageCodeHeader
+    },
+    data
+})
+
+export const dispatchWithAlert = (action, data) => store.dispatch({
+    ...action,
+    meta: {
+        ...action.meta,
+        ...showAlert(data)
+    }
+})
+
+export const actionsPending = (types) => (typeof types === 'string' ? [types] : types).reduce((x, type) => {
```

Comment:
do I understand correctly that `.reduce` is used here as https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some?

Reply:
done